utils/index.js

```diff
@@ -208,3 +208,67 @@ module.exports.processMarkdown = require('markdown-it')({
   linkify: true,
   typographer: true
 })
+
+/**
+ * Returns an array of express Router instances
+ * or []
+ */
+module.exports.loadThemeRoutes = function (app) {
+  console.log('Loading configured theme routes...')
+  const config = require('../config')
+  const path = require('path')
```

Comment:
@starsinmypockets maybe we can move requiring built-in modules to the top of the file?